packages/autocomplete/src/elements/Multiselect.js

```diff
@@ -0,0 +1,449 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { MenuView, Item } from '@zendeskgarden/react-menus';
+import { TextGroup, Label, FauxInput, Input, Hint, Message } from '@zendeskgarden/react-textfields';
+import { Tag, Close } from '@zendeskgarden/react-tags';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { FieldContainer, KEY_CODES } from '@zendeskgarden/react-selection';
+import AutocompleteContainer from '../containers/AutocompleteContainer';
+
+const VALIDATION = {
+  SUCCESS: 'success',
+  WARNING: 'warning',
+  ERROR: 'error',
+  NONE: 'none'
+};
+
+const StyledNoItemsMessage = styled.div`
+  margin: 16px;
+  text-align: center;
+`;
+
+const StyledMenuOverflow = styled.div`
+  overflow-y: auto;
+  max-height: ${props => props.maxHeight || 'inherit'};
+`;
+
+const StyledInput = styled(Input)`
+  && {
+    flex-grow: 1;
+    margin: 2px;
+    width: 60px;
```

Comment:
Should this be `flex-basis: 60px` since I assume this is meant to be a minimum width kind of thing?

Reply:
Correct. I also had some specificity weirdness that I've corrected as well.